requirements.prod.txt

```diff
@@ -10,11 +10,19 @@ chardet==3.0.4
     # via requests
 idna==2.10
     # via requests
+opensafely-pipeline @ https://github.com/opensafely-core/pipeline/archive/refs/heads/support-job-runner.zip
```

Comment:
Is the plan to merge this into pipeline before landing this?

Reply:
Yes, I'll merge the changes to pipeline, and then update the version here so we're always running from a tag 👍